doc/admin-guide/files/records.config.en.rst

```diff
@@ -1326,6 +1326,24 @@ HTTP Redirection
    This setting determines the maximum size in bytes of uploaded content to be
    buffered for HTTP methods such as POST and PUT.
 
+.. ts:cv:: CONFIG proxy.config.http.redirect_loopback_mode INT 0
+   :reloadable:
+
+   This setting determines how redirects should be handled when the target of
+   the redirect is a loopback address.
+
+   ===== ======================================================================
+   Value Description
+   ===== ======================================================================
+   ``0`` The redirect is not followed. It is returned to the client.
+   ``1`` The redirect is not followed. A 403 is returned to the client.
+   ``2`` The redirect is followed. **Use this setting with caution!**
+   ===== ======================================================================
+
+.. warning:: Following a redirect to a loopback address can be dangerous, as it allows the controller of an origin to arrange a probe the Trafficserver host. Generally, an administrator should never configure Trafficserver to follow these redirects.
+
+The default behavior is to return the origin response to the client.
+
```

Comment:
Note to reviewers:

<img width="996" alt="screen shot 2018-08-22 at 00 52 02" src="https://user-images.githubusercontent.com/905298/44445318-188f5180-a5a6-11e8-8bc2-c9987e553f56.png">

This is a rendering in my browser. The big idea was that an administrator who uses the dangerous setting is fully informed of the risk, and the reader knows the default behavior is reasonable.

Reply:
This config has been changed to `redirect_enabled` and the doc has been expanded and updated since this screenshot.